core/search/multiterm_query.cpp

```diff
@@ -159,17 +159,16 @@ doc_iterator::ptr multiterm_query::execute(
     ++it;
```

Comment:
Can you add assert to this and prev usage it?
```C++
assert(it != end(iters));
*it;
```

Reply:
done